python/TestHarness/testers/CSVDiff.py

```diff
@@ -71,6 +73,12 @@ def processResultsCommand(self, moose_dir, options):
             if self.specs.isValid('override_abs_zero'):
                 csvdiff.append('--custom-abs-zero %s' % (' '.join(self.specs['override_abs_zero'])))
 
+            if self.specs.isValid('ignore_columns'):
+                csvdiff.append('--ignore-fields %s' % (' '.join(self.specs['ignore_columns'])))
+
+            if self.specs.isValid('custom_columns'):
```

Comment:
let s check the arguments passed to make sure they make sense:
an ignored column for example should not be a custom or an override column

Reply:
unless CSVDiff already errors on this nicely